### PR TITLE
[CTOR-96] Rubrik - Enhance jobs and cache modes

### PR DESCRIPTION
**Pull request by itoussies**
## Description

Update of the monitoring procedure following the plugin and pack enhancement.

## Target version (i.e. version that this PR changes)

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] Cloud
- [X] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rubrik-restapi.md [hunk 1]
```diff
@@ -537,39 +540,41 @@ Les options disponibles pour chaque modèle de services sont listées ci-dessous
...
+| --filter-job-name           | Filter jobs by job name.                                                                                                                                           |
+| --filter-job-type           | Filter jobs by job type.                                                                                                                                           |
+| --filter-object-type        | Filter jobs by object type.                                                                                                                                        |
+| --filter-location-name      | Filter jobs by location name.                                                                                                                                      |
+| --unit                      | Select the time unit for last execution time thresholds. May be's' for seconds, 'm' for minutes, 'h' for hours, 'd' for days, 'w' for weeks. Default is seconds.   |
+| --limit                     | Define the number of entries to retrieve for the pagination(default: 500).                                                                                         |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-rubrik-restapi.md:561]:
```suggestion
| --limit                     | Define the number of entries to retrieve for the pagination (default: 500).                                                                                         |
```

File: pp/integrations/plugin-packs/procedures/applications-rubrik-restapi.md [hunk 2]
```diff
@@ -531,39 +534,41 @@ All available options for each service template are listed below:
...
+| --filter-job-name           | Filter jobs by job name.                                                                                                                                           |
+| --filter-job-type           | Filter jobs by job type.                                                                                                                                           |
+| --filter-object-type        | Filter jobs by object type.                                                                                                                                        |
+| --filter-location-name      | Filter jobs by location name.                                                                                                                                      |
+| --unit                      | Select the time unit for last execution time thresholds. May be's' for seconds, 'm' for minutes, 'h' for hours, 'd' for days, 'w' for weeks. Default is seconds.   |
+| --limit                     | Define the number of entries to retrieve for the pagination(default: 500).                                                                                         |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-rubrik-restapi.md:555]:
```suggestion
| --limit                     | Define the number of entries to retrieve for the pagination (default: 500).                                                                                         |
```

File: pp/integrations/plugin-packs/procedures/applications-rubrik-restapi.md [hunk 3]
```diff
@@ -179,7 +179,7 @@ yum install centreon-pack-applications-rubrik-restapi
 </Tabs>
 
 2. Whatever the license type (*online* or *offline*), install the **Rubrik Rest API** connector through
-the **Configuration > Monitoring Connector Manager** menu.
+the **Configuration > Monitoring Connectors Manager** menu.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-rubrik-restapi.md:182]:
```suggestion
the **Configuration > Monitoring Connector Manager** menu.
```

**Review by cg-tw — DISMISSED** [review]

**Comment by omercier** [comment]
@cg-tw be cautious, this PR is bound to staging but it is related to the 24.02 release

**Review by omercier — APPROVED** [review]

**Review by smau-centreon — APPROVED** [review]

**Pull request merged**